src/components/Footer/index.js

```diff
@@ -61,13 +93,27 @@ const Footer = () => (
               </li>
             </ul>
           </Col>
+
           <Col md={4} sm={12} xs={12} className={styles.links}>
-            <p className="monserrat">
-              Quran.com ( also known as The Noble Quran, Al Quran, Holy Quran, Koran ){' '}
-              is a pro bono project.
+            <p className={styles.header}>
+              <LocaleFormattedMessage
+                id={'local.selectLabel'}
+                defaultMessage={'Site Language'}
+              />
             </p>
 
-            <p className="monserrat">&copy; QURAN.COM. ALL RIGHTS RESERVED 2016</p>
+            <div className={styles.list}>
+              <LocaleSwitcher/>
+
+              <p className="monserrat">
+                <LocaleFormattedMessage
+                  id={'nav.aboutQuranProject'}
+                defaultMessage={'Quran.com (also known as The Noble Quran, Al Quran, Holy Quran, Koran) is a pro bono project.'}
+                />
+              </p>
+
+              <p className="monserrat">&copy; QURAN.COM. ALL RIGHTS RESERVED 2016</p>
```

Comment:
Yep, we need to translate this and other static pages. Next PR, it was already became huge.